### PR TITLE
[frontend] - sort currency list

### PR DESCRIPTION
**Pull request by puckpuck**
## Changes

Sorts the list of currencies in the currency selector.
Screen Shot 2022-10-14 at 2 34 20 PM

### CODE REVIEW

**Review by reyang — APPROVED** [review]
Awesome!

File: src/frontend/providers/Currency.provider.tsx [hunk 1]
```diff
@@ -36,13 +36,23 @@ const CurrencyProvider = ({ children }: IProps) => {
     SessionGateway.setSessionValue('currencyCode', currencyCode);
   }, []);
 
+  const currencyCodeList = currencyCodeListUnsorted.sort((n1, n2) => {
```
Comment (mic-max) [inline src/frontend/providers/Currency.provider.tsx:39]: I'm not sure what `currencyCodeListUnsorted` is seems just like an array of strings though. `Array.sort()` should suffice in that case.

**Review by mic-max — APPROVED** [review]
LGTM nice change, 1 comment about implementation detail

File: src/frontend/providers/Currency.provider.tsx (hunk 1 above)
Comment (puckpuck) [reply src/frontend/providers/Currency.provider.tsx]: When I wrote the code, I felt like I was doing something unnatural... and yes `sort()` is all I needed without having to define actual compare logic.

**Pull request merged**